app/controllers/crate/security.js

```diff
@@ -0,0 +1,21 @@
+import Controller from '@ember/controller';
+import { service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+export default class SecurityController extends Controller {
+  @service releaseTracks;
+  @service sentry;
+
+  @tracked crate;
+  @tracked data;
+
+  constructor() {
+    super(...arguments);
+    this.reset();
+  }
+
+  reset() {
+    this.crate = undefined;
+    this.data = undefined;
+  }
+}
```

Comment:
```suggestion
```

Since we explicitly assign the controller fields in the `setupController()` fn in the route, I don't think we need an explicit controller implementation at all and can just rely on the automatically generated empty one.